apps/api/v2/src/ee/event-types/event-types_2024_06_14/event-types.repository.ts

```diff
@@ -1,38 +1,20 @@
+import { DatabaseEventType } from "@/ee/event-types/event-types_2024_06_14/services/output-event-types.service";
 import { PrismaReadService } from "@/modules/prisma/prisma-read.service";
 import { PrismaWriteService } from "@/modules/prisma/prisma-write.service";
 import { UsersService } from "@/modules/users/services/users.service";
 import { UserWithProfile } from "@/modules/users/users.repository";
 import { Injectable } from "@nestjs/common";
+import { EventType, User, Schedule } from "@prisma/client";
 
-import {
-  getEventTypeById,
-  transformBookingFieldsApiToInternal,
-  transformLocationsApiToInternal,
-} from "@calcom/platform-libraries";
-import { CreateEventTypeInput_2024_06_14 } from "@calcom/platform-types";
+import { getEventTypeById } from "@calcom/platform-libraries";
+import { InputEventTransformed_2024_06_14 } from "@calcom/platform-types";
 import type { PrismaClient } from "@calcom/prisma";
 
-type InputEventTransformed = Omit<
-  CreateEventTypeInput_2024_06_14,
-  | "lengthInMinutes"
-  | "locations"
-  | "bookingFields"
-  | "bookingLimitsCount"
-  | "onlyShowFirstAvailableSlot"
-  | "bookingLimitsDuration"
-  | "offsetStart"
-  | "periodType"
-  | "periodDays"
-  | "periodCountCalendarDays"
-  | "periodStartDate"
-  | "periodEndDate"
-  | "recurrence"
-> & {
-  length: number;
-  slug: string;
-  locations?: ReturnType<typeof transformLocationsApiToInternal>;
-  bookingFields?: ReturnType<typeof transformBookingFieldsApiToInternal>;
+type EventTypeWithRelations = EventType & {
```

Comment:
We have this type already in `apps/api/v2/src/ee/event-types/event-types_2024_06_14/services/output-event-types.service.ts`

Reply:
https://github.com/calcom/cal.com/commit/85b59dbdd844ce45ffe869146dc10d3baa77415e